VideoWeb/VideoWeb/ClientApp/src/app/services/conference/participant.service.ts

```diff
@@ -42,7 +42,7 @@ export class ParticipantService {
         return this._endpointParticipants;
     }
 
-    private _virtualMeetingRooms: VirtualMeetingRoomModel[];
+    private _virtualMeetingRooms: VirtualMeetingRoomModel[] = [];
```

Comment:
This fixed an error that was been thrown when the virtual meeting rooms where not loaded and is unrelated to the other changes.